.github/workflows/tests.yaml

```diff
@@ -3,8 +3,30 @@ name: Tests
 on:
   push:
     branches: [ master ]
+    paths:
+      - '.github/workflows/tests.yaml'
+      - 'requirements*.txt'
+      - 'setup.py'
+      - 'setup.cfg'
+      - 'MANIFEST.in'
+      - 'pyproject.toml'
+      - 'pep-508-install'
+      - 'scripts/**'
+      - 'rasterio/**'
+      - 'tests/**'
```

Comment:
I'm not sure about this, maybe we want to run GH actions any time we have push to master 🤷‍♂️

Reply:
Sometimes unrelated issues may arise by running on merge to master. I don't think it would be a bad thing to enable.